### PR TITLE
chore: made `chainConfig` files match testnet configuration

### PR DESCRIPTION
**Pull request by Torres-ssf**
# Summary

This PR updates the `chainConfig` files in our tests to align with the latest testnet upgrade, ensuring consistency across the testing environment. The `gasCosts` new values resulted in expressively reduced transaction fees, requiring some fine-tuning of a few tests.

# Checklist

- [x] All **changes** are **covered** by **tests** (or not applicable)
- [x] All **changes** are **documented** (or not applicable)
- [x] I **reviewed** the **entire PR** myself (preferably, on GH UI)
- [x] I **described** all **Breaking Changes** (or there's none)

### CODE REVIEW

**Review by arboleya — APPROVED** [review]

**Review by nedsalk — APPROVED** [review]

File: .fuel-core/configs/chainConfig.json [hunk 1]
```diff
@@ -145,154 +145,154 @@
...
+            "LightOperation": {
+              "base": 3957,
+              "units_per_gas": 48
             }
           }
         }
       },
       "base_asset_id": "0xf8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07",
-      "block_gas_limit": 100000000,
+      "block_gas_limit": 30000000,
```
Comment (maschad) [inline .fuel-core/configs/chainConfig.json:295]: Same here

File: .fuel-core/configs/chainConfig.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
           "max_inputs": 255,
           "max_outputs": 255,
           "max_witnesses": 255,
-          "max_gas_per_tx": 100000000,
+          "max_gas_per_tx": 30000000,
```
Comment (maschad) [inline .fuel-core/configs/chainConfig.json:10]: Why is this set to `30000000` as opposed to `100000000` as in the [config](https://github.com/FuelLabs/chain-configuration/blob/master/upgradelog/ignition-dev/consensus_parameters/6.json)?

File: .fuel-core/configs/chainConfig.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
           "max_predicate_length": 102400,
           "max_predicate_data_length": 102400,
           "max_message_data_length": 102400,
-          "max_gas_per_predicate": 100000000
+          "max_gas_per_predicate": 30000000
```
Comment (maschad) [inline .fuel-core/configs/chainConfig.json:20]: Same here

**Review by maschad — COMMENTED** [review]
Nice work @Torres-ssf , I assume you are using [this config](https://github.com/FuelLabs/chain-configuration/blob/2dbbe4c6b40f9e458d65945a1f6f1e25888cf669/upgradelog/ignition-dev/consensus_parameters/6.json) just a few questions about the discrepancies

File: .fuel-core/configs/chainConfig.json (hunk 2 above)
Comment (Torres-ssf) [reply .fuel-core/configs/chainConfig.json:10]: @maschad I am using [this consensus parameters](https://github.com/FuelLabs/chain-configuration/blob/2dbbe4c6b40f9e458d65945a1f6f1e25888cf669/upgradelog/ignition/consensus_parameters/5.json) instead, which is from the latest upgrade for testnet, not for devnet.

**Review by petertonysmith94 — APPROVED** [review]
Nice one @Torres-ssf.

I think it'd be worth documenting this process - which _could_ then serve as a basis of automating down the line.

**Review by danielbate — APPROVED** [review]

**Review by maschad — APPROVED** [review]

**Pull request merged**